### DISCUSSION TITLE
Redirect Global Query Parameters

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#10353] [Feature Requests]
Adding global redirects parameters, which instead of an API call responding with a 2xx or 4xx status code, does a 3xx redirect to a specified url.

The use case is to use this in html forms without javascript.

Example:
```
<form 
   action="https://directus.example.com/auth/login?redirect[200]=https://www.example.com/checkout&redirect[400]=https://www.example.com/login/error" 
   method="post"
>
  <label for="name">Email</label>
  <input id="name" type="email" name="email">
  
  <label for="name">Password</label>
  <input id="name" type="password" name="password">
  
  <input type="submit" value="Login">
</form>
```

**Comment by rijkvanzanten** [comment]
Oooh that's a great idea! Also like the fact that you allow a different URL for different status codes

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
